### ISSUE TITLE
Find a alternative for Pandas in Haskell

### ISSUE THREAD

**Issue by NickSeagull** [DataHaskell/OrganizationTasks#7]
We might even have to develop one

**Comment by stites** [comment]
I think the most active data frames implementation can be found at [acowley/Frames](https://github.com/acowley/Frames).

**Comment by NickSeagull** [comment]
Yes @stites, the only "painful" fact is that when dealing with datasets with a lot of columns (for example 279 for the Arrhythmia Dataset from UCI) it would be really tedious to type all of them. So something dynamic-ish like Pandas would be awesome

**Comment by richardfergie** [comment]
Frames can use template haskell to avoid typing column names (see http://acowley.github.io/Frames/#outline-container-orgheadline5).

Not perfect, because it requires column headers to be present in the csv file

**Comment by NickSeagull** [comment]
Actually there is the https://github.com/chrisdone/labels library which exactly allows this

**Comment by yamad** [comment]
I've done exactly zero data frame work in Haskell, but it's worth mentioning that the R community has a newish library that improves how data frames work in R: https://github.com/hadley/tibble . For evaluating a data frame API, looking at the interface changes they made driven by experience seems like a good idea. In particular, tibbles have more intuitive subsetting operators that the base R data frames, in my opinion.
